hermes_eea/calibration/calibration.py

```diff
@@ -152,12 +149,34 @@ def parse_l0_sci_packets(data_filename: Path) -> dict:
     log.info(f"Parsing packets from file:{data_filename}.")
 
     pkt = ccsdspy.FixedLength.from_file(
-        os.path.join(hermes_eea._data_directory, "EEA_sci_packet_def.csv")
+        os.path.join(hermes_eea._data_directory, "hermes_EEA_sci_packet_def.csv")
     )
     data = pkt.load(data_filename)
     return data
 
 
+def converting_ccsds_times_to_cdf(coarse, fine):
```

Comment:
@Alrobbertz do we need this or does the HERMESData container already provide this functionality?

Reply:
I'll delete this one as it is just something Liam used temporarily, but I use another one under util/time, so let me know